src/main/java/ca/ciccc/silverBullet/controller/GameManger.java

```diff
@@ -0,0 +1,9 @@
+package ca.ciccc.silverBullet.controller;
+
+import FileInput.FileInput;
+import javafx.scene.media.AudioClip;
+
+public class GameManger {
```

Comment:
Typo

Reply:
I suppose you should change this signature to AbstractController class.
Then move this class into the controller package.